source/shared/cpp/ObjectModel/ActionParserRegistration.h

```diff
@@ -7,47 +7,50 @@
 
 namespace AdaptiveCards
 {
-    class BaseActionElement;
-    class ParseContext;
+class BaseActionElement;
```

Comment:
nit: is this something agreed to change? looks different from what we had before.

Reply:
yes. but it's open to change. it's controlled by NamespaceIndentation. the default is none, and inner is used here. to add spaces, it has to be set to All. You can see the difference from https://clang.llvm.org/docs/ClangFormatStyleOptions.html